client/core/errorstrings.cpp

```diff
@@ -2,70 +2,74 @@
 
 using namespace amnezia;
 
-QString errorString(ErrorCode code){
+QString errorString(ErrorCode code) {
+    QString errorMessage;
+
     switch (code) {
 
     // General error codes
-    case(NoError): return QObject::tr("No error");
-    case(UnknownError): return QObject::tr("Unknown Error");
-    case(NotImplementedError): return QObject::tr("Function not implemented");
+    case(NoError): errorMessage = QObject::tr("No error");
```

Comment:
It is necessary to add 'break' to each 'case', otherwise 'switch' will be executed to the end and all error messages will have the text "Internal error"